File: app/src/main/java/com/packg/easynotes/Elements/TextNote.java
package com.packg.easynotes.Elements;

import java.util.ArrayList;

public class TextNote extends Element{
    protected String name;
    protected String text;

    public TextNote(int id, String name,String text){
        this.id = id;
        this.name = name;
        this.text = text;
        this.type = "TextNote";
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getName() {
        return name;
    }


}

File: app/src/main/java/com/packg/easynotes/Elements/TextNote.kt
package com.packg.easynotes.Elements

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "text_note_table")
class TextNote(@ColumnInfo(name = "name") val name : String,
               @ColumnInfo(name = "text") val text : String) : Element() {

    @ColumnInfo(name = "id")
    @PrimaryKey(autoGenerate = true) var id: Long = 0

}